kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RollingUpdateProcessServiceIntegrationTest.java

```diff
@@ -309,8 +306,8 @@ public void testContainerIdAsAlias() throws Exception {
 
         assertClientException(() -> processClient.startProcess(CONTAINER_ALIAS, PROCESS_ID_EVALUATION_2),
                 404,
-                MessageFormat.format(PROCESS_DEFINITION_NOT_FOUND, PROCESS_ID_EVALUATION_2, CONTAINER_ALIAS),
-                MessageFormat.format(NO_PROCESS_AVAILABLE_WITH_ID, PROCESS_ID_EVALUATION_2));
+                "Could not find process definition \"" + PROCESS_ID_EVALUATION_2 + "\" in container \"" + CONTAINER_ALIAS + "\"",
+                "No process available with given id : " + PROCESS_ID_EVALUATION_2);
```

Comment:
why this has been changed?

Reply:
It was changed because the message was imported from kie-server-rest-jbpm. To make the tests independent on rest/service dependencies (as they aren't available in container test runs) I replaced it with strings. Not the cleanest solution TBH, I intended to make build fixed fast.